internal/common/common.go

```diff
@@ -7,14 +7,14 @@ import (
 	"encoding/json"
 	"encoding/xml"
 	"fmt"
+	"github.com/go-errors/errors"
+	"github.com/privacybydesign/gabi/big"
+	"github.com/sirupsen/logrus"
```

Comment:
Hi David, me and Sietse discussed to redo this PR to handle moving scheme directories the same way as files, i.e., in the current folder. This means the file cannot is very likely on the same filesystem and we can use `os.rename`. It also means we disregard the `conf.TempPath` and we end up with an atomic operation (no half scheme left behind in case of errors). Does someone know if `os` offers a more robust way to do this in general (i.e., rollback if something has gone wrong)?

As for the imports, I forgot to run my formatter :)

Reply:
Ah ok, in that case i would suggest we mark this PR as a draft to indicate it shouldn't be merged